docs/testing/index.md

```diff
@@ -0,0 +1 @@
+# Testing Documentation
```

Comment:
if we have this file in docs/testing why do we need a .gitkeep as well?

Reply:
We don't 😄 I added the .gitkeep in the first commit, and this in a later one, will remove.

Reply:
Resolved with 248ec55184ec71104acf701aa5fe0008135ecfe6